script/publish_persistent_udfs

```diff
@@ -25,6 +26,8 @@ from bigquery_etl.parse_udf import (
 
 UDF_RE = re.compile(r"udf_(?:js_)?([a-zA-z0-9_]+)")
 
+OPTIONS_LIB_RE = re.compile(r"library = \"gs://[^\\\"]+/([^\\\"]+)\"")
```

Comment:
perhaps I'm missing the goal of some of these backslashes, but
- raw strings make backslashes literal in python, even if you were trying to escape a quote
- there's no point in refusing to match `\` a gcs path because it's already not allowed
```suggestion
OPTIONS_LIB_RE = re.compile(r'library = "gs://[^"]+/([^"]+)"')
```